src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/ConnectivityTest.cs

```diff
@@ -44,6 +45,8 @@ public static void EnvironmentHostNameTest()
                     sessionSpid = reader.GetInt16(0);
                 }
 
+                Assert.Equal(sqlclientSPID, sessionSpid);
```

Comment:
xunit's Assert.Equal puts "expected" before "actual"

Reply:
Yes I changed that :)